documentation/modules/kafkaconnectors/proc-enabling-kafkaconnectors.adoc

```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies:
+//
+// assembly-deployment-configuration-kafka-connect.adoc
+
+[id='proc-enabling-kafkaconnectors-{context}']
+= Enabling `KafkaConnector` resources
+
+To enable `KafkaConnectors` for a Kafka Connect cluster, add the `strimzi.io/use-connector-resources` annotation to the `KafkaConnect` or `KafkaConnectS2I` custom resource. When enabled, connectors in the Kafka Connect cluster are created and managed using `KafkaConnectors`.
```

Comment:
connectors in the Kafka Connect cluster are created and managed using `KafkaConnectors`.  -- but only when we've deployed a KafkaConnector resource?